### DISCUSSION TITLE
Absolute positioning of image in reveal.js presentation

### DISCUSSION THREAD

**Discussion by lg2011** [quarto-dev/quarto-cli#1160] [Q&A] [answered]
In the documentation (section Advanced Reveal) it is written "The absolute class lets you position elements at arbitrary positions on a slide.". However, I can see no implementation of such a feature in the documentation of reveal.js (4.3.1) and could not get it to work. Is there a specific extension / plugin in Quarto to provide this feature ?

**Comment by cderv** [comment, marked as answer]
You should just be able to use the example in the doc here: https://quarto.org/docs/presentations/revealjs/advanced.html#absolute-position

Here is an adaption using only image
````markdown
---
title: test
format: revealjs
---

## Cats

![](https://images.pexels.com/photos/617278/pexels-photo-617278.jpeg){.absolute top=200 left=0 width="350" height="300"}

![](https://images.pexels.com/photos/45201/kitty-cat-kitten-pet-45201.jpeg){.absolute top=50 right=50 width="450" height="250"}

![](https://images.pexels.com/photos/104827/cat-pet-animal-domestic-104827.jpeg){.absolute bottom=0 right=50 width="300" height="300"}
````

![image](https://user-images.githubusercontent.com/6791940/174038267-e84b764e-6704-474e-ae0d-e9389ffa861a.png)

What is not working for you ? Can you share more details ? Reproducible example ? Versions you are using ?
